python/application-load-balancer/app.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+import base64
+from aws_cdk import (
+    aws_autoscaling as autoscaling,
+    aws_ec2 as ec2,
+    aws_elasticloadbalancingv2 as elbv2,
+    core,
+)
+
+
+class LoadBalancerStack(core.Stack):
+    def __init__(self, app: core.App, id: str) -> None:
+        super().__init__(app, id)
+
+        vpc = ec2.Vpc(self, "VPC")
+
+        data = open("./httpd.sh", "rb").read()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the resource returned by the following method call: *open*. The resource is allocated by call *builtins.open*. Execution paths that do not contain closure statements were detected. To prevent this resource leak, close the object returned by *open()* in a try-finally block or declare it using a `with` statement.

**More info**
[View details about the `with` statement in the Python developer&#39;s guide](https://www.python.org/dev/peps/pep-0343/) (external link).